### PR TITLE
v6 minimap patch: updated exports, df calls, and added corrupted spac…

### PR DESCRIPTION
**Pull request by Bind**
…e type

### CODE REVIEW

File: content/utilities/mini-map/plugin.js [hunk 1]
```diff
@@ -158,7 +158,9 @@ class Plugin {
               data.push({
                 x: i,
                 y: j,
-                type: df.spaceTypeFromPerlin(df.getPerlin({x: i, y: j}))
+                type: df.entityStore.spaceTypeFromPerlin(
```
Comment (phated) [inline content/utilities/mini-map/plugin.js:161]:
```suggestion
                type: df.spaceTypeFromPerlin(
```
Looks like `spaceTypeFromPerlin` is still on the `df` object

File: content/utilities/mini-map/plugin.js [hunk 2]
```diff
@@ -175,6 +177,8 @@ class Plugin {
             ctx.fillStyle = '#24247d'; // outer nebula
           } else if (data[i].type === 2) {
             ctx.fillStyle = '#000000'; // deep space
+          } else if (data[i].type === 3) {
+            ctx.fillStyle = "magenta"; // Corrupted... Will sub out with actual color
```
Comment (phated) [inline content/utilities/mini-map/plugin.js:181]: I mined a chunk and color-pickered it
```suggestion
            ctx.fillStyle = "#250025"; // Corrupted
```

**Review by phated — CHANGES_REQUESTED** [review]
You are amazing ❤️

**Comment by phated** [comment]
@Bind Would you be able to drop in your screenshot too? Love to see that corrupted space in the screenshot!

**Comment by solsticesolutions** [comment]
Only big issue I had was it being larger than the game window so I had to zoom out my chrome window to get the title bar back to close/manipulate it

**Comment by Bind** [comment]
> Only big issue I had was it being larger than the game window so I had to zoom out my chrome window to get the title bar back to close/manipulate it

I tweaked the default window size to hopefully mitigate this!

**Review by phated — APPROVED** [review]
Looking great! I pushed a metadata change that is going to bump this to the top and I'll send a website update that shows the version numbers in a minute.

**Pull request merged**